ios/Classes/MapboxMapFactory.swift

```diff
@@ -139,12 +139,13 @@ class MapboxMapFactory: NSObject, FlutterPlatformViewFactory {
         var mapInitOptions = MapInitOptions()
         var eventTypes = [String]()
         var pluginVersion = ""
+        var channelSuffix = 0
```

Comment:
just curious that for Android I don't see initing channelSuffix with 0, why is the implementation different?

Reply:
This is to make swiftlint happy - it's complaining to forsecasting. Value (and others as well) is always passed within this map